test/integration/test_create_dss_docker_image.py

```diff
@@ -16,6 +18,22 @@
 from exasol.ds.sandbox.lib import pretty_print
 
 
+class StderrCapture:
```

Comment:
Currently unused, see [comment](https://github.com/exasol/data-science-sandbox/issues/59#issuecomment-1814510146) in ticket #59.